### ISSUE TITLE
about fine-tuning the loss function 

### ISSUE THREAD

**Issue by doomysday** [Zheng222/IDN-Caffe#21]
In the paper, the author say "we first train the network with MAE loss and then **fine-tune** it by MSE loss.",but I cannot find the clue about how to fine-tune the Loss function in the training phase, anybody know where it is in the code?
